swift-sdk/Internal/RequestCreator.swift

```diff
@@ -425,12 +432,14 @@ struct RequestCreator {
         if let packageName = Bundle.main.appPackageName {
             args[JsonKey.InApp.packageName] = packageName
         }
-
+        
         return .success(.get(createGetRequest(forPath: Const.Path.getRemoteConfiguration, withArgs: args as! [String: String])))
     }
     
     // MARK: - PRIVATE
     
+    private let authFailedMessage = "Both email and userId are nil"
```

Comment:
I think static would be better since this has no instance specific information. So basically
`private static let authMissingMessage` and in other places `Self.authMissingMessage`. (notice capital S).